internal/internal_task_pollers.go

```diff
@@ -467,7 +464,12 @@ func (latp *localActivityTaskPoller) PollTask() (interface{}, error) {
 
 func (latp *localActivityTaskPoller) ProcessTask(task interface{}) error {
 	result := latp.handler.executeLocalActivityTask(task.(*localActivityTask))
-	latp.laTunnel.deliverResult(result)
+	select {
```

Comment:
Add comment explaining as this code is confusing. The way I understood is that laResultCh is synchronous and there is guarantee that doneCh is closed if nobody is listening on laResultCh.

Reply:
your understanding is correct. Will add more comments.